### DISCUSSION TITLE
How to save my current pipeline/settings in case of a reboot/crash?

### DISCUSSION THREAD

**Discussion by pborah88** [Stability-AI/StableSwarmUI#294] [Q&A]
I have my desired community variant of SD model loaded along with a refiner model of my choice. I have everything dialed in just the right amount from type of samplers, scheduler, sampling steps, cfg, refiner control, upscaling, attention guidance, height multiplier, custom VAEs, etc. to get the kind of generations I want. How do I save these settings (similar to how I can save a workflow in ComfyUI)? I'm finding myself having to reboot the server every 200-250 generations when it starts slowing down/freezing,  and each time I have to reconfigure all these parameters all over again. It completely forgets it's previous state prior to a manual shutdown. The only thing it retains are the prompts.

**Comment by mcmonkey4eva** [comment]
So, A: as long as your browser is modern and functional, your Generate parameters are saved (at least temporarily) across restarts/reloads.
B: at the bottom of the Generate tab, click "Presets", and "Create New Preset". You can save any parameters you want in a preset, and load/apply them at any time. Name your preset `Default` if you want it to be used for the default parameters any time load you load (eg a new browser, you delete your browser history, or you hit the "Reset Params to Default" quicktool)
